src/components/molecules/EntranceStepsInputFieldSet/EntranceStepsInputFieldSet.tsx

```diff
@@ -31,35 +37,93 @@ export const EntranceStepsInputFieldSet: React.FC<EntranceStepsInputFieldSetProp
   errors,
   index,
   name,
+  onUpdate,
   onRemove,
   register,
 }) => {
+  const fieldButtons = `buttons`;
+  const fieldUrl = `videoUrl`;
+  const fieldTemplate = `template`;
+  const fieldset = `${name}[${index}]`;
+  const inputButtons = `${fieldset}${fieldButtons}`;
+  const inputUrl = `${fieldset}${fieldUrl}`;
+  const inputTemplate = `${fieldset}${fieldTemplate}`;
+
+  const handleRemove = useCallback(() => onRemove({ index }), [
+    onRemove,
+    index,
+  ]);
+
+  const handleChange: ChangeEventHandler<HTMLInputElement> = useCallback(
```

Comment:
Could we extract out this code to remove the duplication?
This code is in EntranceButtonsInputFieldSet.tsx  as well. And it's in a few other places too

Reply:
It can be DRY-ed up or even replaced with Redux, now that ti's obvous what the problem is. For next PR though